l10n_it_reverse_charge/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Davide Corio
+# Copyright 2017 Alex Comba - Agile Business Group
+# Copyright 2017 Lorenzo Battistini - Agile Business Group
+# Copyright 2017 Marco Calcagni - Dinamiche Aziendali srl
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'ITA - Reverse Charge IVA',
+    'version': '12.0.1.0.0',
+    'category': 'Localization/Italy',
+    'summary': 'Reverse Charge for Italy',
```

Comment:
Dato che "name" e "summary" non sono traducibili meglio lasciarli in italiano.

Reply:
Tradotti @primes2h, nel summary ho tolto la parte 'per l'Italia' perché mi sembra ridondante